spec/examples/validations/css.spec.js

```diff
@@ -30,6 +30,14 @@ describe('css', function() {
     `)
   );
 
+  it('allows valid filter', () =>
+      assertPassesValidation(css, `
```

Comment:
I hate to nitpick here, but the indentation convention in the project is two spaces—here and a few other places you’ve got four. Mind updating? Thanks/sorry!